modules/create-wif-cluster-cli.adoc

```diff
@@ -124,35 +128,34 @@ Once the WIF is configured, the following service accounts, roles, and groups ar
 - compute.viewer
 
 |cloud-credential-operator-gcp-ro-creds
-|cloud_credential_operator_gcp_ro_creds_v{product-version}
+|cloud_credential_operator_gcp_ro_creds_v<y-stream-version>
 
 |openshift-cloud-network-config-controller-gcp
-|openshift_cloud_network_config_controller_gcp_v{product-version}
+|openshift_cloud_network_config_controller_gcp_v<y-stream-version>
 
 |openshift-gcp-ccm
-|openshift_gcp_ccm_v{product-version}
+|openshift_gcp_ccm_v<y-stream-version>
 
 |openshift-gcp-pd-csi-driver-operator
 |- compute.storageAdmin
 - iam.serviceAccountUser
 - resourcemanager.tagUser
-- openshift_gcp_pd_csi_driver_operator_v{product-version}
+- openshift_gcp_pd_csi_driver_operator_v<y-stream-version>
 
 |openshift-image-registry-gcp
-|openshift_image_registry_gcs_v{product-version}
+|openshift_image_registry_gcs_v<y-stream-version>
 
 |openshift-ingress-gcp
-|openshift_ingress_gcp_v{product-version}
+|openshift_ingress_gcp_v<y-stream-version>
 
 |openshift-machine-api-gcp
-|openshift_machine_api_gcp_v{product-version}
+|openshift_machine_api_gcp_v<y-stream-version>
 
 |Access via SRE group:sd-sre-platform-gcp-access
 |sre_managed_support
-
 |===
 
-For further details about WIF configuration roles and their assigned permissions, see link:https://github.com/openshift/managed-cluster-config/blob/master/resources/wif/4.18/vanilla.yaml[managed-cluster-config].
+For the complete list of WIF configuration roles and their assigned permissions, see link:https://github.com/openshift/managed-cluster-config/blob/master/resources/wif/4.19/vanilla.yaml[managed-cluster-config].
```

Comment:
@AedinC This link is specific to v4.19, would it be possible to make it dynamic to the y-stream-version that this documentation points to? Otherwise, the parent of this link has the list of all the y-stream versions we support:
https://github.com/openshift/managed-cluster-config/tree/master/resources/wif

I raise this to as we may forget to update this link in later versions, resulting in its pointing to data which would not be useful to the customer.